.github/workflows/build-debug-node.yml

```diff
@@ -0,0 +1,54 @@
+name: Build debug node 
+
+on: 
+  workflow_dispatch:
+    inputs:
+      version:
+        required: true
+        description: 'Node.js version'
+
+jobs:
+  build:
+    name: Build Debug version of Node.js
+    runs-on: buildjet-4vcpu-ubuntu-2204
+    strategy:
+      fail-fast: false
+    steps:
+      - name: Install dependencies 
+        run: apt-get install python3 g++ make python3-pip
+
+      - name: Download Node.js source
+        uses: actions/checkout@v4
+        with: 
+          repository: 'nodejs/node'
+          ref: 'v${{ github.event.inputs.version }}'
+          path: 'nodejs'
+
+      - name: Configure nodejs with debug flag 
+        run: ./configure --debug
+        working-directory: 'nodejs'
+
+      - name: Compile the nodejs
+        run: make -j4
```

Comment:
Is it possible to use `make -j$(nproc --all)` here to default to the number of available cores?